### ISSUE TITLE
Error running on multiple images in a row - workaround with TF session context manager

### ISSUE THREAD

**Issue by AsbjornOlling** [justinshenk/fer#1]
Hi there

Wonderful library - I really enjoy the api design - makes it really easy (and fun) to use. Good work :+1: 

But I did run into one (minor) issue:

It doesn't work if running on more than one discrete image in a row.

The first image analyzed with `FER().detect_emotions(img)` always works fine, but running the function again with a different image results in a tensorflow error (paraphrased, I lost the original): "Tensorflow Tensor is not element of this graph".

I'm not sure exactly what went wrong (I'm not too familiar with tensorflow), but wrapping the call in a `with tf.Session():` context manager resolved the issue. 

I hope you'll look into fixing this in the library, as it was really enjoyable to use (apart from this haunting bug).

**Comment by justinshenk** [comment]
Thanks for your feedback and post @AsbjornOlling! I wasn't able to reproduce the issue with the following modification to `example.py`:

```python
image = cv2.imread("justin.jpg")
result = detector.detect_emotions(image)
print(result)
result = detector.detect_emotions(cv2.imread('justin2.png'))
print(result)
```
This runs without error on my system. If you can provide the code that caused this error please let me know.

**Comment by AsbjornOlling** [comment]
I didn't notice that you responded until just now.

This is the relevant code snippet, which won't run without the tensorflow context manager.
It's contained inside a function which just takes a `bytes` object as input.

It's within a Flask microservice, so I can't load the images as a file, like you do.

I'm using python 3.6, if that should make any difference.

```python
    assert isinstance(img, bytes), f"Img is not bytes. Is: {type(img)}"

    # read image bytes to opencv matrix
    img = cv2.imdecode(
        np.asarray(bytearray(img), dtype=np.uint8),
        cv2.IMREAD_COLOR
    )

    # analyze image with FER
    with tf.Session():
        result = FER().detect_emotions(img)
    assert result != [], "No faces found in image."
```

Thank you for looking into the issue :+1:

**Comment by justinshenk** [comment]
I didn't realize this bug was within Flask. I had issues as well due to threading and Flask being single-threaded (see [issue](https://github.com/keras-team/keras/issues/2397)). For a Flask project [deepemotion.io](https://github.com/justinshenk/deepemotion/blob/master/app.py) I did the following (using the Video class in combination):

```
detector = FER(emotion_model=os.environ.get('EMOTION_API_URL', None))
current_video = None
...
@app.route('/analyze', methods=['GET'])
def analyze():
        global current_video
        ...
        df = current_video.analyze(
            detector,
            display=False, ...)
```
Glad to know about the tf.Session() solution!

**Comment by justinshenk** [comment]
Closing issue.

**Comment by justinshenk** [comment]
Found issue in previous version, fixed in `v19.0.2`.
